Source/santad/EventProviders/SNTEndpointSecurityFileAccessAuthorizer.mm

```diff
@@ -52,7 +52,7 @@
 // `std::string` is used.
 using PathTargets = std::pair<std::string_view, std::variant<std::string_view, std::string, Unit>>;
 
-const char *kBadCertHash = "BAD_CERT_HASH";
+NSString *kBadCertHash = @"BAD_CERT_HASH";
```

Comment:
Should this not be a const?

Reply:
It could be, but would need to change the types on the cache and the return type of `getCertificateHash`, which would require changes to the call sites (and tests). I took the path of least resistance here since NSString isn't mutable anyways.